class-izooto-integration.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Izooto integration with PWA plugin.
+ */
+class Izooto_Integration {
+
+	public function __construct() {
+
+		add_action( 'wp_front_service_worker', [ $this, 'register_izooto_service_worker_script' ] );
```

Comment:
Use `array()` format because its open source plugin and we it have been used in <5.4 PHP version then it will give fatal error.